### PR TITLE
grafana: supply the explanation on the metrics of Flow-Control in TiKV.

### PR DESCRIPTION
**Pull request by LykxSassinator**
This pr supplements the explanation on the Flow-Control metrics in Tikv-Details part.

### First-time contributors' checklist 

- [x] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-cn) that's required for repo owners to accept my contribution.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [x] master (the latest development version)
- [x] v7.6 (TiDB 7.6 versions)
- [x] v7.5 (TiDB 7.5 versions)
- [x] v7.4 (TiDB 7.4 versions)
- [x] v7.3 (TiDB 7.3 versions)
- [x] v7.1 (TiDB 7.1 versions)
- [x] v6.5 (TiDB 6.5 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: grafana-tikv-dashboard.md [hunk 1]
```diff
@@ -181,6 +181,21 @@ aliases: ['/docs-cn/dev/grafana-tikv-dashboard/','/docs-cn/dev/reference/key-mon
 
 ![TiKV Dashboard - Storage metrics](/media/tikv-dashboard-storage.png)
 
+### Flow Control
+
+- Scheduler flow：每个 TiKV 实例的 scheduler 的实时流量
+- Scheduler discard ratio：每个 TiKV 实例的 scheduler 的请求拒绝比率，大于 0 表示触发流控且 TiKV 实例繁忙
```
Comment (Connor1996) [inline grafana-tikv-dashboard.md:187]: 当 pending compaction bytes 超过阈值时，会根据超过阈值部分的值，按比例线性增加 discard ratio

File: grafana-tikv-dashboard.md [hunk 2]
```diff
@@ -181,6 +181,21 @@ aliases: ['/docs-cn/dev/grafana-tikv-dashboard/','/docs-cn/dev/reference/key-mon
...
+- Throttle duration：由于触发流控后，scheduler 执行请求的阻塞时间，如果存在统计值，则表明存在流控
```
Comment (Connor1996) [inline grafana-tikv-dashboard.md:188]: 当 L0 过多时

File: grafana-tikv-dashboard.md [hunk 3]
```diff
@@ -181,6 +181,21 @@ aliases: ['/docs-cn/dev/grafana-tikv-dashboard/','/docs-cn/dev/reference/key-mon
...
+- Scheduler throttled CF：由于达到流控阈值，触发 RocksDB 限流的 CF
+- Flow controller actions：由于达到流控阈值，触发 RocksDB 限流的原因
+- Flush/L0 flow：每个 TiKV 实例的 RocksDB 的不同 CF 刷写 L0 的实时流量
```
Comment (Connor1996) [inline grafana-tikv-dashboard.md:191]: 不同 CF 的 Flush 流量和 L0 compaction 的流量

File: grafana-tikv-dashboard.md [hunk 4]
```diff
@@ -181,6 +181,21 @@ aliases: ['/docs-cn/dev/grafana-tikv-dashboard/','/docs-cn/dev/reference/key-mon
 
 ![TiKV Dashboard - Storage metrics](/media/tikv-dashboard-storage.png)
 
+### Flow Control
+
+- Scheduler flow：每个 TiKV 实例的 scheduler 的实时流量
+- Scheduler discard ratio：每个 TiKV 实例的 scheduler 的请求拒绝比率，当 Compaction pending bytes 超过阈值时，会根据超过阈值部分的值，按比例线性增加 discard ratio
+- Throttle duration：L0 文件过多并触发流控后，scheduler 执行请求的阻塞时间，如果存在统计值，则表明存在流控
```
Comment (Connor1996) [inline grafana-tikv-dashboard.md:188]:
```suggestion
- Throttle duration：L0 文件过多并触发流控后，scheduler 执行请求的阻塞时间。如果存在统计值，则表明存在流控
```

File: grafana-tikv-dashboard.md [hunk 5]
```diff
@@ -181,6 +181,21 @@ aliases: ['/docs-cn/dev/grafana-tikv-dashboard/','/docs-cn/dev/reference/key-mon
 
 ![TiKV Dashboard - Storage metrics](/media/tikv-dashboard-storage.png)
 
+### Flow Control
+
+- Scheduler flow：每个 TiKV 实例的 scheduler 的实时流量
+- Scheduler discard ratio：每个 TiKV 实例的 scheduler 的请求拒绝比率，当 Compaction pending bytes 超过阈值时，会根据超过阈值部分的值，按比例线性增加 discard ratio
```
Comment (Connor1996) [inline grafana-tikv-dashboard.md:187]:
```suggestion
- Scheduler discard ratio：每个 TiKV 实例的 scheduler 的请求拒绝比率，当 Compaction pending bytes 超过阈值时，会根据超过阈值部分的值，按比例线性增加 discard ratio。被拒绝的请求由客户端重试。如果非 0，则表明存在流控
```

**Review by Connor1996 — COMMENTED** [review]
rest LGTM

File: grafana-tikv-dashboard.md [hunk 6]
```diff
@@ -181,6 +181,21 @@ aliases: ['/docs-cn/dev/grafana-tikv-dashboard/','/docs-cn/dev/reference/key-mon
 
 ![TiKV Dashboard - Storage metrics](/media/tikv-dashboard-storage.png)
 
+### Flow Control
+
+- Scheduler flow：每个 TiKV 实例的 scheduler 的实时流量
+- Scheduler discard ratio：每个 TiKV 实例的 scheduler 的请求拒绝比率，当 Compaction pending bytes 超过阈值时，会根据超过阈值部分的值，按比例线性增加 discard ratio。被拒绝的请求由客户端重试。如果非 0，则表明存在流控
```
Comment (qiancai) [inline grafana-tikv-dashboard.md:187]:
```suggestion
- Scheduler discard ratio：每个 TiKV 实例的 scheduler 的请求拒绝比率。如果该比例不为 0，则表明存在流控。当 Compaction pending bytes 超过阈值时，TiKV 会根据超过阈值部分的值，按比例线性增加 Scheduler discard ratio。被拒绝的请求需要由客户端重试
```

File: grafana-tikv-dashboard.md [hunk 7]
```diff
@@ -181,6 +181,21 @@ aliases: ['/docs-cn/dev/grafana-tikv-dashboard/','/docs-cn/dev/reference/key-mon
...
+- Throttle duration：L0 文件过多并触发流控后，scheduler 执行请求的阻塞时间。如果存在统计值，则表明存在流控
+- Scheduler throttled CF：由于达到流控阈值，触发 RocksDB 限流的 CF
```
Comment (qiancai) [inline grafana-tikv-dashboard.md:189]: CF 是什么的简称呢

File: grafana-tikv-dashboard.md [hunk 8]
```diff
@@ -181,6 +181,21 @@ aliases: ['/docs-cn/dev/grafana-tikv-dashboard/','/docs-cn/dev/reference/key-mon
...
+- Flow controller actions：由于达到流控阈值，触发 RocksDB 限流的原因
+- Flush/L0 flow：每个 TiKV 实例的 RocksDB 的不同 CF 的 Flush 流量和 L0 compaction 的流量
```
Comment (qiancai) [inline grafana-tikv-dashboard.md:191]:
```suggestion
- Flush/L0 flow：每个 TiKV 实例上 RocksDB 的不同 CF 的 Flush 流量和 L0 compaction 的流量
```

File: grafana-tikv-dashboard.md [hunk 9]
```diff
@@ -181,6 +181,21 @@ aliases: ['/docs-cn/dev/grafana-tikv-dashboard/','/docs-cn/dev/reference/key-mon
...
+- Flow control factors：触发 RocksDB 限流相关的因素
+- Compaction pending bytes：每个 TiKV 实例的 RocksDB 的实时等待 compaction 的大小
```
Comment (qiancai) [inline grafana-tikv-dashboard.md:193]:
```suggestion
- Compaction pending bytes：每个 TiKV 实例上 RocksDB 实时等待 compaction 的数据的大小
```

File: grafana-tikv-dashboard.md [hunk 10]
```diff
@@ -181,6 +181,21 @@ aliases: ['/docs-cn/dev/grafana-tikv-dashboard/','/docs-cn/dev/reference/key-mon
...
+- Txn command throttled duration：由于限流，与事务相关的命令的阻塞时间，正常情况下，均为 0
```
Comment (qiancai) [inline grafana-tikv-dashboard.md:194]:
```suggestion
- Txn command throttled duration：由于限流，与事务相关的命令的阻塞时间，正常情况下，该指标为 0
```

File: grafana-tikv-dashboard.md [hunk 11]
```diff
@@ -181,6 +181,21 @@ aliases: ['/docs-cn/dev/grafana-tikv-dashboard/','/docs-cn/dev/reference/key-mon
...
+- Non-txn command throttled duration：由于限流，其他命令的阻塞时间，正常情况下，均为 0
```
Comment (qiancai) [inline grafana-tikv-dashboard.md:195]:
```suggestion
- Non-txn command throttled duration：由于限流，其他命令的阻塞时间，正常情况下，该指标为 0
```

File: grafana-tikv-dashboard.md (hunk 8 above)
Comment (qiancai) [reply grafana-tikv-dashboard.md:191]: 太多“的”影响阅读

File: grafana-tikv-dashboard.md [hunk 12]
```diff
@@ -181,6 +181,21 @@ aliases: ['/docs-cn/dev/grafana-tikv-dashboard/','/docs-cn/dev/reference/key-mon
 
 ![TiKV Dashboard - Storage metrics](/media/tikv-dashboard-storage.png)
 
+### Flow Control
+
+- Scheduler flow：每个 TiKV 实例的 scheduler 的实时流量
+- Scheduler discard ratio：每个 TiKV 实例的 scheduler 的请求拒绝比率。如果该比例不为 0，则表明存在流控。当 Compaction pending bytes 超过阈值时，TiKV 会根据超过阈值部分的值，按比例线性增加 Scheduler discard ratio。被拒绝的请求需要由客户端重试
```
Comment (Connor1996) [inline grafana-tikv-dashboard.md:187]: “需要由” 感觉会给用户困扰，认为需要客户做些什么

**Review by qiancai — APPROVED** [review]

File: grafana-tikv-dashboard.md [hunk 13]
```diff
@@ -181,6 +181,21 @@ aliases: ['/docs-cn/dev/grafana-tikv-dashboard/','/docs-cn/dev/reference/key-mon
 
 ![TiKV Dashboard - Storage metrics](/media/tikv-dashboard-storage.png)
 
+### Flow Control
+
+- Scheduler flow：每个 TiKV 实例的 scheduler 的实时流量
+- Scheduler discard ratio：每个 TiKV 实例的 scheduler 的请求拒绝比率。如果该比例不为 0，则表明存在流控。当 Compaction pending bytes 超过阈值时，TiKV 会根据超过阈值部分的值，按比例线性增加 Scheduler discard ratio。被拒绝的请求将自动由客户端重试
+- Throttle duration：L0 文件过多并触发流控后，scheduler 执行请求的阻塞时间。如果存在统计值，则表明存在流控
```
Comment (Oreoxmt) [inline grafana-tikv-dashboard.md:188]:
```suggestion
- Throttle duration：L0 文件过多并触发流控后，scheduler 执行请求的阻塞时间。如果存在统计数据，则表明存在流控
```

File: grafana-tikv-dashboard.md [hunk 14]
```diff
@@ -181,6 +181,21 @@ aliases: ['/docs-cn/dev/grafana-tikv-dashboard/','/docs-cn/dev/reference/key-mon
...
+- Scheduler throttled CF：由于达到流控阈值，触发 RocksDB 限流的 CF
+- Flow controller actions：由于达到流控阈值，触发 RocksDB 限流的原因
+- Flush/L0 flow：每个 TiKV 实例上 RocksDB 的不同 CF 的 Flush 流量和 L0 compaction 的流量
+- Flow control factors：触发 RocksDB 限流相关的因素
+- Compaction pending bytes：每个 TiKV 实例上 RocksDB 实时等待 compaction 的数据的大小
+- Txn command throttled duration：由于限流，与事务相关的命令的阻塞时间，正常情况下，该指标为 0
+- Non-txn command throttled duration：由于限流，其他命令的阻塞时间，正常情况下，该指标为 0
```
Comment (Oreoxmt) [inline grafana-tikv-dashboard.md:195]:
```suggestion
- Txn command throttled duration：由于限流，与事务相关的命令的阻塞时间。正常情况下，该指标为 0
- Non-txn command throttled duration：由于限流，非事务相关的命令的阻塞时间。正常情况下，该指标为 0
```

**Review by Oreoxmt — APPROVED** [review]

File: grafana-tikv-dashboard.md [hunk 15]
```diff
@@ -181,6 +181,21 @@ aliases: ['/docs-cn/dev/grafana-tikv-dashboard/','/docs-cn/dev/reference/key-mon
 
 ![TiKV Dashboard - Storage metrics](/media/tikv-dashboard-storage.png)
 
+### Flow Control
+
+- Scheduler flow：每个 TiKV 实例的 scheduler 的实时流量
+- Scheduler discard ratio：每个 TiKV 实例的 scheduler 的请求拒绝比率。如果该比例不为 0，则表明存在流控。当 Compaction pending bytes 超过阈值时，TiKV 会根据超过阈值部分的值，按比例线性增加 Scheduler discard ratio。被拒绝的请求将自动由客户端重试
```
Comment (qiancai) [inline grafana-tikv-dashboard.md:187]:
```suggestion
- Scheduler discard ratio：每个 TiKV 实例的 scheduler 的请求拒绝比率。如果该比例大于 0，则表明存在流控。当 Compaction pending bytes 超过阈值时，TiKV 会根据超过阈值部分的值，按比例线性增加 Scheduler discard ratio。被拒绝的请求将自动由客户端重试
```

**Review by Connor1996 — APPROVED** [review]

**Pull request merged**